test-suite/output/Arguments_renaming.out

```diff
@@ -84,8 +84,8 @@ Argument lists should agree on the names they provide.
 The command has indeed failed with message:
 Sequences of implicit arguments must be of different lengths.
 The command has indeed failed with message:
-Argument number 3 is a trailing implicit, so it can't be declared non
-maximal. Please use { } instead of [ ].
+Argument z is a trailing implicit, so it can't be declared non maximal.
```

Comment:
Maybe we should have a test where we don't name the argument so it still has to print `number 123`

Reply:
Good remark, done.